examples/Fortran/CMakeLists.txt

```diff
@@ -0,0 +1,23 @@
+include_directories("${CMAKE_CURRENT_BINARY_DIR}/../../src/serialbox-fortran")
+link_directories("${CMAKE_CURRENT_BINARY_DIR}/../../src/serialbox-fortran")
```

Comment:
@thfabian: Can you have a look at the CMake config whether we need the include or not.